Week3/string.js

```diff
@@ -0,0 +1,27 @@
+// Strings Question 1
+
+//  Consider the following string:
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+// Q: Add the string to your file and log it.
+
+// Solution
+
+console.log(myString);
+
+// Log the length of myString.
+
+// Solution : we use length property for the solution.
+
+console.log(myString.length);
+
+// Q:The commas make that the sentence is quite hard to read. Find a way to remove the commas from the string and replace them with spaces.
+
+// Solution
+
+const myString = 'hello,this,is,a,difficult,to,read,sentence';
+
+let regex = /,/gi;
```

Comment:
Clear! But I think you've redefined `myString` and because you already defined it on line `5` (and it's a `const`) so I don't think this will work properly

Reply:
You are completely rigth, i change the string name , so it works now.